Sources/Datadog/DatadogCore/Storage/Files/File.swift

```diff
@@ -30,6 +30,9 @@ internal protocol ReadableFile {
     /// Reads the available data in this file.
     func read() throws -> Data
 
+    /// Creates InputStream for reading the available data from this file.
+    func readStream() throws -> InputStream
+
```

Comment:
By switching to `readStream()` in `FileReader` below:
```diff
- let events = try decode(data: file.read())
+ let events = try decode(stream: file.readStream())
```
it sees we have no other use for `read()` method and it can be removed entirely. WDYT @maxep? (we can do it here on in a follow-up PR after merge)

Reply:
It's used a lot by unit tests and it's definitely an easier to use API for that case.

Reply:
Yes, let's keep it for now 👍